orchestra/bots/tests/test_staffbot.py

```diff
@@ -208,3 +218,41 @@ def test_restaff_command_errors(self):
         self.assertEquals(response.get('text'),
                           (bot.task_assignment_does_not_exist_error
                            .format(worker.user.username, task.id)))
+
+    def test_get_staffing_request_messsage(self):
+        def _task_factory(status, path):
+            description_no_kwargs = {'path': path}
+            return TaskFactory(
+                status=status,
+                step=StepFactory(
+                    slug='stepslug',
+                    description='the step',
+                    detailed_description_function=description_no_kwargs),
+                project=ProjectFactory(
```

Comment:
i use this specific text to verify that the email gets formatted appropriately.  leaving as-is :)